cluster-provision/k8s/1.17/provision.sh

```diff
@@ -68,7 +68,7 @@ name=Kubernetes
 baseurl=http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
```

Comment:
try this url https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
before you change gpg check